content/docs/user-guide/experiment-management/sharing-experiments.md

```diff
@@ -4,64 +4,14 @@ In a regular Git workflow, <abbr>DVC repository</abbr> versions are typically
 synchronized among team members. And [DVC Experiments] are internally connected
 to this commit history, so you can similarly share them.
 
-## Basic workflow: store as peristent commits
+[dvc experiments]: /doc/user-guide/experiment-management
 
-The most straightforward way to share experiments is to store them as
-[persistent](/doc/user-guide/experiment-management/persisting-experiments) Git
-commits and share them like you would [any other commit]. This will share all
-code and metadata associated with the experiment.
-
-```cli
-$ dvc exp branch quare-zips my-branch
-Git branch 'my-branch' has been created from experiment 'quare-zips'.
-To switch to the new branch run:
-        git checkout my-branch
+## Method 1: Less steps
```

Comment:
Other ideas:

- Less steps, less flexible
- For developers
- Stay within the Experiments UI